CP1404_practicals/prac_08/taxi_simulator.py

```diff
@@ -0,0 +1,20 @@
+"""
+Create taxi simulator that utilizes
+ Taxi and SilverServiceTaxi classes.
+"""
+
+from prac_08.car import Car
+from prac_08.taxi import Taxi
+from prac_08.silver_service_taxi import SilverServiceTaxi
+
+MENU = "q)uit, c)hoose taxi, d)rive"
+
+
+def main():
+    """Create a taxi simulator program that utilizes Taxi and SilverServiceTaxi classes.
+    This program will loop through until the user quites."""
+    print("Lets drive!")
+    print(MENU)
```

Comment:
You could condense these two lines using string formatting eg. print("Lets drive!\n{}".format(MENU))

Reply:
thanks for the advice I will make the changes.